common/static/js/vendor/tinymce/BUILD_README.txt

```diff
@@ -1,20 +1,29 @@
-Instructions for creating js/tinymce.full.min.js
+# Instructions for updating tinymce to a newer version:
 
-1. Ensure that the dependencies (NodeJS, Jake, and other dependencies) are installed. If necessary,
-   install them per the directions on https://github.com/tinymce/tinymce/tree/4.0.20.
-2. Unzip edx-platform/vendor_extra/tinymce/JakePackage.zip into this directory (so that Jakefile.js resides in this directory).
-3. Clean install the dependencies that were unzipped
-   npm ci
-4. Run the following command in the tinymce directory:
-   npx jake clean-js
-5. Run the following command in the tinymce directory:
-   npx jake minify bundle[themes:*,plugins:*]
-6. Cleanup by deleting the Unversioned files that were created from unzipping jake_package.zip.
+1. Download the desired version from https://github.com/tinymce/tinymce/releases
+2. If it’s a major update, follow the official migration doc and update the codebase as needed.
+3. Setup the codemirror-plugin as per the instruction below.
+4. Find all the EDX specific changes in the currently used version of tinymce by searching for the string "EDX" in the vendor/js/tinymce dir.
+5. Merge the EDX specific changes with the newly downloaded version.
+6. Follow the instructions given below to create the new version of js/tinymce.full.min.js
 
-Instructions for updating tinymce to a newer version:
+# Instruction for setting codemirror-plugin
 
-1. Download the desired version from https://github.com/tinymce/tinymce/releases
-2. Find all the EDX specific changes that were made to the currently used version of tinymce by searching for
-   the string "EDX" in this directory.
-3. Merge the EDX specific changes with the new version.
-4. Follow the instructions above for creating the new version of js/tinymce.full.min.js
+1. Download the tinymce-codemirror-plugin from https://gitlab.com/tinymce-plugins/tinymce-codemirror
+2. Open terminal in the downloaded plugin directory and run the following commands:
+  ```
+   npm install
+   npm run prepublish (This command will generate the minified file in the plugin directory)
+  ```
+3. Remove the tinymce-codemirror/plugins/codemirror/codemirror-4.8 directory
+4. Move the tinymce-codemirror/plugins directory to common/static/js/vendor/tinymce/js/plugins/ directory.
+
+# Instructions for creating js/tinymce.full.min.js
+
+1. Follow the README file of the downloaded tinymce’s codebase and install all the requirements.
+2. Open terminal and change directory to the newly downloaded tinymce.
+3. Run the build command: “yarn build”, this will create multiple zip files in the build directory.
+4. Unzip the tinymce_<version>.zip file in common/static/vendor/js/
+5. Unzip vendor_extra/tinymce/JakePackage.zip in  common/static/vendor/js/tinymce/.
```

Comment:
> Also, is there a specific reason to use js/vendor/tinymce/js/tinymce instead of just js/vendor/tinymce?

the `common/static/js/vendor/` is the open-edx structure for storing vendor files where as `js/tinymce/` is the tinymce's structure (the js dir contains other files like README, packge.json etc.).

Reply:
> I believe it's common/static/js/vendor/tinymce/,

Yeah, that's true! I've updated the instructions.

Reply:
> where as js/tinymce/ is the tinymce's structure

I couldn't find any references to this structure in TinyMCE or CodeMirror, but I see that it's referenced like this in multiple places within the edx-platform, so let's keep it as-is for backward compatibility.

> (the js dir contains other files like README, packge.json etc.).

I don't see other files in [vendor/tinymce/js](https://github.com/openedx/edx-platform/tree/9201eb2029128c2849dfc67eafa89afde83f2f8b/common/static/js/vendor/tinymce/js), but I suppose that keeping [BUILD_README.txt](https://github.com/openedx/edx-platform/blob/9201eb2029128c2849dfc67eafa89afde83f2f8b/common/static/js/vendor/tinymce/BUILD_README.txt) in a separate directory increases its visibility :+1: